dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -42,6 +42,7 @@
          -->
     <CppWinRTParameters>-ignore_velocity</CppWinRTParameters>
     <CppWinRTOptimized>true</CppWinRTOptimized>
+    <CppWinRTHeapEnforcement>true</CppWinRTHeapEnforcement>
```

Comment:
Is there any documentation on what flags/options are available for cppWinrt ?

Reply:
They should be documented here - https://github.com/microsoft/cppwinrt/tree/master/nuget - but I see that this one isn't...

Reply:
Good question. I think it's mostly all just on by default. -optimized is the thing we have to opt into and I think this would be on by default except for the XAML bug. @Scottj1s @kennykerr are there any others we should know about/turn on?